src/receive_data.py

```diff
@@ -0,0 +1,18 @@
+# Lint as: python3
+"""
+Receive data from R script
```

Comment:
"Receive data from standard input in CSV format". Doesn't really matter where it's coming from.

Reply:
Changed.